test/e2e/storage/csi_mock_volume.go

```diff
@@ -31,7 +31,7 @@ import (
 	"google.golang.org/grpc/status"
 	v1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
-	storagev1alpha1 "k8s.io/api/storage/v1alpha1"
+	storagev1beta1 "k8s.io/api/storage/v1beta1"
```

Comment:
The feature tag should be removed so it always runs

Reply:
Funny, this also occured to me this morning during breakfast :sweat_smile: 

Done.